docs/content/latest/yugabyte-platform/administer-yugabyte-platform/ldap-authentication.md

```diff
@@ -88,4 +101,8 @@ For additional information, see [Update a configuration key](https://yugabyte.st
 
 ## Define the Yugabyte Platform Role
 
-You need to define a Yugabyte Platform-specific role for each user on your LDAP server by setting the a `YugabytePlatformRole` annotation.
+You need to define a Yugabyte Platform-specific role for each user on your LDAP server by setting the `YugabytePlatformRole` annotation. The value set for this annotation is read during the Yugabyte Platform login. Note that if the value is modified on the LDAP server, the change is propagated to Yugabyte Platform and automatically updated during login. Password updates are also automatically handled.
+
+If the role is not specified, users are created with ReadOnly privileges by default, which can be modified by the local super admin.
+
+When LDAP is set up on a Windows Active Directory (AD) server, the user is expected to have permissions to query the AD's properties from that server. If the permissions have not been granted, Yugabyte Platform defaults its role to ReadOnly, which can later be modified by the local super admin.
```

Comment:
"the user is expected to query its properties"
Replace "AD's properties" -> "user's properties"

Rest everything looks good

Reply:
changed AD's to user's